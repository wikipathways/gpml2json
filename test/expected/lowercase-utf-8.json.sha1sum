b9427ddcef1f56d3680c665c00253e573289e671 ?-
