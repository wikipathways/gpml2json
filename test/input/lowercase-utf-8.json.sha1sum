ab569421cd58a9f7fc23f38b9b434d3d6689807a ?-
