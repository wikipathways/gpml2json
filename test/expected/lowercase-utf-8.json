{"pathway":{"@context":["https://cdn.rawgit.com/wikipathways/WpVocabularies/7a46a05/contexts/pvjs.jsonld"],"backgroundColor":"white","borderWidth":0,"color":"black","contains":["e3838"],"drawAs":"rect","gpmlElementName":"Pathway","height":140.57708161582852,"kaavioType":"Group","name":"New Pathway","padding":5,"type":["Pathway"],"width":227.4608408903545,"x":0,"y":0,"zIndex":0,"fontSize":12,"fontWeight":"bold","textAlign":"left","verticalAlign":"top","dataSourceVersion":"20170828","standardName":"New Pathway","displayName":"New Pathway","textContent":"Name: New Pathway","isSimilarTo":"http://wikipathways.org/index.php/Special:SearchPathways?query=New%20Pathway&doSearch=1","id":"hash://sha256/912d64d91a51efd199df4077ca3429d2f5e7bab565ac790d456791ecefa6025c"},"entityMap":{"e3838":{"textContent":"GeneProduct","id":"e3838","wpType":"GeneProduct","x":117.46084089035449,"y":90.57708161582852,"width":80,"height":20,"zIndex":32768,"fontSize":10,"verticalAlign":"middle","textAlign":"center","color":"#000000","backgroundColor":"#ffffff","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"drawAs":"rect","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["DataNode","SingleFreeNode","GeneProduct"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"}}}
